contents.json

```diff
@@ -19263,6 +19263,29 @@
       ],
       "date_added": "Apr 17 2020",
       "suggested_by": "@dkhamsing"
+    },
+    {
+      "title": "palette-from-image",
+      "category-ids": [
+        "color",
+        "clone"
+      ],
+      "description": "Inspire from https://earthview.withgoogle.com",
```

Comment:
Could you change the description to: Inspired by https:..

Reply:
Updated. Thanks